source/Missions/Services/Arena/ArenaTestGameManager.cs

```diff
@@ -48,7 +48,6 @@ public void StartGame()
             else
             {
                 ScreenManager.PopScreen();
-                // TODO add popup message
```

Comment:
This is already done

Reply:
I don't understand this comment, this line was removed. Could you clarify if I'm misunderstanding

Reply:
Oops